PROTO_tests/tests/pdarray_creation_test.py

```diff
@@ -206,6 +206,10 @@ def test_randint_num_dtype(self, dtype):
 
     @pytest.mark.parametrize("size", pytest.prob_size)
     def test_randint_misc(self, size):
+
+        # Test that int_scalars covers uint8, uint16, uint32
+        ak.randint(low=np.uint8(1), high=np.uint16(100), size=np.uint32(100))
```

Comment:
I see several tests like this, where it instantiates an array without checking the output.  Is it just to test there is no error thrown?  I think we should also assert the output is reasonable.

Reply:
Honestly, I don't know.  It was in nearly all of the non-PROTOs tests, and was missing from some of the PROTOs ones.  In the interests of ensuring that PROTOs covers the same conditions that non-PROTOs does, I included them where they were missing.

Reply:
OK, I see.  Sorry, I should have read the ticket description more closely.